pkg/util/utils.go

```diff
@@ -305,6 +304,21 @@ func GetImageConfig(changes []string) (ImageConfig, error) {
 	return config, nil
 }
 
+// Parse and validate a signal name or number
+func ParseSignal(rawSignal string) (syscall.Signal, error) {
+	// Strip off leading dash, to allow -1 or -HUP
+	basename := strings.TrimPrefix(rawSignal, "-")
+
+	signal, err := signal.ParseSignal(basename)
+	if err != nil {
+		return -1, err
+	}
+	if 1 <= signal && signal <= 64 { // FIXME: hardcoding SIGRTMAX
```

Comment:
I don't see `SIGRTMAX` in the x/sys/unix library, which makes me think that it's not defined anywhere in official Go libraries. We may have to leave it here.